system/otherLoadTest/playlist.xml

```diff
@@ -22,7 +22,8 @@
 		<variations>
 			<variation>NoOptions</variation>
 		</variations>
-		<command>$(SYSTEMTEST_CMD_TEMPLATE) $(CUSTOM_TARGET);\
+		<command>[ $(JDK_VERSION) -gt 8 ] $(AND_IF_SUCCESS) ADD_OPENS=$(Q)--add-opens java.base/java.lang=ALL-UNNAMED --add-opens java.base/java.io=ALL-UNNAMED$(Q); \
```

Comment:
I would prefer to have this defined as a variable in https://github.com/AdoptOpenJDK/openjdk-tests/blob/master/system/systemtest.mk, so that it can be used by other tests. For example, both MiniMix_5m and system_custom can use it. And we can merge MiniMix_5m and MiniMix_jdk8_5m into one test target.

Reply:
@llxia : good point! I've made the update in the PR.  

- Tested with JDK11 with `system_custom`: Internal/Grinder/14424/ (green)
- Test for JDK8 with `system_custom`: Internal/Grinder/14427 (waiting to be scheduled) 
- Test for JDK11 with `MiniMix_5m` target: Internal/Grinder/14426 (waiting to be scheduled)